packages/sdk/src/reliability-score.ts

```diff
@@ -162,6 +181,13 @@ export default class ReliabilityScore {
           peerId
         );
       } else {
+        // This should be (successful - failed / sent )
```

Comment:
if this is an indication of a future change pls open an issue and remove the comment

Reply:
this was an idea but i don't think it will be needed. will remove